js/renovation/ui/scroll_view/scrollbar.tsx

```diff
@@ -411,7 +411,10 @@ export class Scrollbar extends JSXComponent<ScrollbarPropsType>() {
     const contentSizeChanged = this.props.contentSize !== this.prevContentSize;
     const containerSizeChanged = this.props.containerSize !== this.prevContainerSize;
 
-    if (contentSizeChanged || containerSizeChanged) {
+    if (this.props.containerDimensionsIsNotZero
```

Comment:
Let's make it the first line of method:
```
if (!this.props.containerDimensionsIsNotZero){
  return;
}
```
Or rename it:
```
if (this.props.containerHasSize){
  return;
}
```
It's very easy to understand what this condition means.

Do we need the same condition in other methods?